tezos_interop/tezos_interop.re

```diff
@@ -556,19 +556,21 @@ module Fetch_storage: {
     };
   };
 
+  // TODO: stop hard coding this
+  let command = "node";
```

Comment:
This  doesn't look right -  the temp file will disappear after the `with_temp_file` finishes . Does this PR intend to keep it for entire life time of the node?

Reply:
No it doesn't, the temp file is kept there, the `with_temp_file` here is above keeping the temp_file open, not about keeping it present. This is already how it's done for other scripts, otherwise the scripts wouldn't work.

Reply:
https://ocsigen.org/lwt/5.2.0/api/Lwt_io#VALwith_temp_file

> When the promise returned by f is resolved, with_temp_file closes the channel and deletes the temporary file by calling Lwt_unix.unlink. Since 3.2.0

Reply:
This was fixed today.

Reply:
Oh then we're abusing of a bug, that's nice, will fix both branches

Reply:
Nope, I was wrong, I'm not using with_temp_file on the other code, mb

Reply:
thank you fixed it